src/map/utils/charutils.cpp

```diff
@@ -7000,6 +7000,73 @@ namespace charutils
         }
     }
 
+    void ReadFishingHistory(CCharEntity* PChar)
+    {
+        TracyZoneScoped;
+
+        if (PChar == nullptr)
+        {
+            return;
+        }
+
+        auto Query = "SELECT fish_catches, fish_linescast, fish_reeled, fish_longest, fish_longest_id, fish_heaviest, fish_heaviest_id "
+                     "FROM char_fishing_history "
+                     "WHERE charid = (?);";
+
+        auto ret = db::preparedStmt(Query, PChar->id);
```

Comment:
The new db system returns `rset` (result set), the old system returns `ret` (return code)